zebra-network/src/lib.rs

```diff
@@ -88,3 +88,7 @@ pub use crate::{
 pub mod types {
     pub use crate::{meta_addr::MetaAddr, protocol::types::PeerServices};
 }
+
+use std::sync::atomic::AtomicBool;
+/// A flag to indicate if zebrad is shutting down.
+pub static IS_SHUTDOWN: AtomicBool = AtomicBool::new(false);
```

Comment:
Nitpick: Zebra is in the process of shutting down, so let's make the flag name clearer 
```suggestion
pub static IS_SHUTTING_DOWN: AtomicBool = AtomicBool::new(false);
```